packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts

```diff
@@ -402,6 +407,18 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
                             normalizeImageSelection(previousSelectedImage);
                         }
 
+                        if (this.initialEditingInfo && this.imageEditInfo) {
+                            // Finish editing, check if image is changed and skip undo snapshot if not
+                            const changeState = checkEditInfoState(
```

Comment:
We already call checkEditInfoState inside applyChange, can we make applyChange return the the value of checkEditInfoState? then we would not need to call checkEditInfoState again, just use the result of applyChange.